.github/workflows/tests.yaml

```diff
@@ -86,10 +90,10 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        os: [macos-latest]
-        python-version: ['3.9', '3.10']
         include:
-          - os: ubuntu-latest
+          - os: macos-latest
+            python-version: '3.10'
+          - os: windows-latest
```

Comment:
As we already have Linux tests with the Docker tests, it would be more interesting to test Windows here. Not sure how easy this is though.